From 3e1dc3cdd07bbe3806970d3aaa2e0d64bef73ea0 Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Mon, 3 Oct 2011 13:06:33 +0200
Subject: [PATCH] axi_ethernet: Remove section mismatch

Warning log:
MODPOST vmlinux.o
WARNING: vmlinux.o(.data+0xb078): Section mismatch in reference
from the variable axienet_of_driver to the function
.init.text:axienet_probe()
The variable axienet_of_driver references
the function __init axienet_probe()
If the reference is valid then annotate the
variable with __init* or __refdata (see linux/init.h) or name the variable:
*_template, *_timer, *_sht, *_ops, *_probe, *_probe_one, *_console,

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/net/ethernet/xilinx/xilinx_axienet_main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
index ad7c938..27da15f 100644
--- a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
@@ -1784,7 +1784,7 @@ static void axienet_dma_err_handler(unsigned long data)
  * device. Parses through device tree and populates fields of
  * axienet_local. It registers the Ethernet device.
  **/
-static int __init
+static int __devinit
 axienet_of_probe(struct platform_device *op)
 {
 	struct device_node *np;
-- 
1.7.0.4

